console/src/channel/external.ts

```diff
@@ -13,6 +13,7 @@ import { Layout } from "@/layout";
 export * from "@/channel/Create";
 export * from "@/channel/ontology";
 export * from "@/channel/palette";
+export * from "@/channel/link";
```

Comment:
I ran into a problem where I could not get the console to build if I put `export * from "@/channel/link"; into where line 14 currently is.